setup.py

```diff
@@ -38,6 +38,7 @@
     'gitversion',
     'APScheduler==3.0.3',
     'python-oembed == 0.2.1',
+    'html5lib<0.99999999'
```

Comment:
Did you get any error?

The version of `html5lib` installed without this line added is `0.9999999` (7 x 9). It matches the requirements from bleach (8 x 9).
0.9999999 < 0.99999999 == true

So I still don't understand why we need to install html5lib specifically.